index.html

```diff
@@ -1,75 +1,425 @@
+              scale data, where Cartesian assumptions of spherical data
+              traditionally break down. &#160;<a href=
+              "http://scitools.org.uk/cartopy/docs/latest/index.html"
+                 target="_blank">Learn more...</a></p>
             </div>
           </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="mbr-section content4 title-banner">
+    <div class="container">
+      <div class="media-container-row">
+        <div class="title col-12 col-md-8">
+          <h2 class="align-center pb-3 mbr-fonts-style display-2">
+            Additional Tools
+            <a class="anchor" name="additional_tools" id="additional_tools"></a>
+          </h2>
+          <h3 class=
+          "mbr-section-subtitle align-center mbr-light mbr-fonts-style display-5">
+          There are several other packages being developed on the SciTools
+          GitHub. Some of the more prominent ones are listed below.</h3>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="mbr-section article content12 blue-bullet-list">
+    <div class="container">
+      <div class="media-container-row">
+        <div class=
+        "mbr-text counter-container col-12 col-md-8 mbr-fonts-style display-7">
+        <ul>
+            <li>
+            <a class="anchor" name="cf_units" id="cf_units"></a>
+              <strong><a href="https://iris-grib.readthedocs.io/en/latest/"
+               target="_blank"
+               style="font-weight: bold;">&#160;</a><a href=
+               "http://scitools.org.uk/cf_units/index.html"
+               target="_blank"><b>cf_units</b></a> -&#160;</strong>is a
+               wrapper class to support Unidata/UCAR UDUNITS-2, and the
+               netcdftime calendar functionality. Provides units of measure.
+               <strong><a href="http://scitools.org.uk/cf_units/index.html"
+               target="_blank">Learn more...</a></strong></li>
+            <li>
+              <strong><a href="https://iris-grib.readthedocs.io/en/latest/"
+                 target="_blank"><b>iris-grib</b></a> -</strong> provides
+                 functionality for converting between weather and climate
+                 datasets that are stored as GRIB files and Iris cubes.
+                 <a href="https://iris-grib.readthedocs.io/en/latest/"
+                  target="_blank">Learn more...</a>
+            </li>
+            <li>
+              <a href="https://github.com/SciTools/nc-time-axis"
+                  target="_blank"><strong>nc_time_axes</strong></a> - provides
+                  support for netcdftime axis in matplotlib. <a href=
+                  "https://github.com/SciTools/nc-time-axis"
+                  target="_blank">Learn more...</a>
+            </li>
+            <li>
+              <a href="https://github.com/SciTools/courses"
+                  target="_blank"><strong>scitools/courses</strong></a> -
```

Comment:
It doesn't really make sense for this to be under the "Additional Tools" section. Although I do agree that it would be nice to be included on the homepage